tests/test_mct_dyn_inflow.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import absolute_import
+import os
+import datetime
+import shutil
+import pytest
+
+from lisfloodutilities.compare.nc import NetCDFComparator
+from lisfloodutilities.compare.pcr import TSSComparator
+
+from lisflood.main import lisfloodexe
+from lisflood.global_modules.settings import LisSettings
+
+from .test_utils import setoptions, mk_path_out
+
+
+class TestInflow():
+
+    case_dir = os.path.join(os.path.dirname(__file__), 'data', 'LF_MCT_UseCase')
+
+    def run(self, date_start, date_end, dtsec, type):
+        # generate inflow (inflow.tss) one pixel upstream of inflow point
+        out_path_ref = os.path.join(self.case_dir, 'reference_mct_dyn')
```

Comment:
I see here and in other unit tests that we are creating output files outside the "out" folder. Is there a reason for that? (I know that sometimes running unit tests in parallel and removing completely the out folder in the teardown method can cause the test to fail: was this the reason? If so I agree we should fix up all teardown methods before fixing the out folder...)